core/src/main/resources/org/apache/spark/ui/static/stagespage-template.html

```diff
@@ -31,10 +31,10 @@ <h4 id="summaryMetricsTitle" class="title-table"></h4>
             </tbody>
         </table>
     </div>
-    <a id="aggregatedMetrics" class="collapse-table">
+    <h4 id="aggregatedMetrics" class="collapse-table">
         <span class="expand-input-rate-arrow arrow-closed" id="arrowtoggle2"></span>
-        <h4 class="title-table">Aggregated Metrics by Executor</h4>
-    </a>
+        <a class="title-table">Aggregated Metrics by Executor</a>
```

Comment:
Dumb question but why is it even a hyperlink?

Reply:
> Dumb question but why is it even a hyperlink?

Highlight and can click  can expand or collapse the table...seems no problem...

Reply:
Ok any other places this needs to happen - is it consistent now?

Reply:
> Ok any other places this needs to happen - is it consistent now?

Yea, not found other place have same issue.